### PR TITLE
Add AndroidSystemFontLister

### PR DESCRIPTION
**Pull request by BobLd**

### CODE REVIEW

**Comment by BobLd** [comment]
@EliotJones now good for review

File: src/UglyToad.PdfPig.Fonts/SystemFonts/SystemFontFinder.cs [hunk 1]
```diff
@@ -89,6 +89,12 @@ static SystemFontFinder()
             {
                 lister = new LinuxSystemFontLister();
             }
+#if NET
```
Comment (EliotJones) [inline src/UglyToad.PdfPig.Fonts/SystemFonts/SystemFontFinder.cs:92]: presumably because it could be mono, or some other reason?

**Review by EliotJones — APPROVED** [review]
Thanks 🎉

File: src/UglyToad.PdfPig.Fonts/SystemFonts/SystemFontFinder.cs (hunk 1 above)
Comment (BobLd) [reply src/UglyToad.PdfPig.Fonts/SystemFonts/SystemFontFinder.cs:92]: This was the less restrictive. The IsAndroid check does not exist for framework (obviously hehe)

**Pull request merged**